apis/config/types.go

```diff
@@ -88,6 +88,8 @@ type TrimaranSpec struct {
 	MetricProvider MetricProviderSpec
 	// Address of load watcher service
 	WatcherAddress string
+	// Interval in seconds for periodic metrics updates. Default is 30 seconds if not set.
+	MetricsUpdateIntervalSeconds int32
```

Comment:
usually optional values are pointers to let disambiguate between "value not set" and "value explicitely set to zero". In this case, does the zero value make sense? do we have the possible ambiguity? if not we can probably use the value (vs the pointer) but the API side will need more consideration.